packages/core/src/config.js

```diff
@@ -59,6 +59,15 @@ export const configSchema = {
       scope: {
         type: 'string'
       },
+      scopeOptions: {
+        type: 'object',
```

Comment:
For all objects we should define valid keys for the same, so that users can understand if they have typos

Reply:
This are valid keys for given option https://github.com/percy/cli/pull/1458/files#diff-e7c1269ddb64f1de14c2c79cac204c16cfc3f8d8535512d218f8e4ad8efc63b6R66-R68. anything else need to be added?